test/teos/conftest.py

```diff
@@ -70,7 +70,12 @@ def setup_node():
             )
         )
         try:
-            btc_addr = bitcoin_cli.getnewaddress()
+            info = bitcoin_cli.getnetworkinfo()
+
+            if info.get("version") >= 210000:
+                bitcoin_cli.createwallet("test_wallet")
+            
+            btc_addr = bitcoin_cli.getnewaddress("test_wallet")
```

Comment:
I think the label here is optional, can you double check?

Reply:
Ah yup runs fine without that. Pushed up that change